src/scrollbar.js

```diff
@@ -88,5 +88,5 @@ ScrollBar.propTypes = {
     onYReachStart: PropTypes.func,
     onYReachEnd: PropTypes.func,
     onXReachStart: PropTypes.func,
-    onXReachEnd: PropTypes.func,
+    onXReachEnd: PropTypes.func
```

Comment:
The trailing comma is recommended by eslint-config-airbnb. Please keep it unchanged.